servicepulse/upgrades/support-policy.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Support Policy
+summary: Versions of ServicePulse that are currently supported
+reviewed: 2022-02-18
+related:
+ - nservicebus/upgrades/release-policy
+ - nservicebus/upgrades/supported-versions
+ - nservicebus/upgrades/supported-platforms
+ - servicecontrol/upgrades/supported-versions
+---
+
+- Major versions are supported for a period of one year after the release of the next major version.
```

Comment:
Same objection as with ServiceInsight. Until we know why we have a 2.x I'm not sure I want to promise that we'll support something non-current. We can always extend additional support later but it's harder to claw it back.